docs/src/main/sphinx/installation/containers.rst

```diff
@@ -1,9 +1,11 @@
-===================
-Trino in containers
-===================
+===========================
+Trino in a Docker container
+===========================
 
 The Trino project provides the `trinodb/trino <https://hub.docker.com/r/trinodb/trino>`_
-Docker image that includes the Trino server and a default configuration.
+Docker image that includes the Trino server and a default configuration. The
+Docker image is published to Docker Hub and can be used with the Docker runtime,
+among several others.
```

Comment:
I don't see why we're adding this sentence. Why not just update the title?

We don't need to specify with runtimes run docker images, just like we don't really define what "a docker image" is.

Reply:
I think it's good to be clear that this isn't the only way to do things / you don't _have_ to follow this guide.

Reply:
I think this is helpful and clarifying and does not do any harm. I would leave as is and merge.